framework/spree/api/utils/create-api-fetch.ts

```diff
@@ -0,0 +1,80 @@
+import { SpreeApiConfig } from '..'
+import { errors, makeClient } from '@spree/storefront-api-v2-sdk'
+import { requireConfigValue } from '../../isomorphic-config'
+import convertSpreeErrorToGraphQlError from '../../utils/convert-spree-error-to-graph-ql-error'
+import type { ResultResponse } from '@spree/storefront-api-v2-sdk/types/interfaces/ResultResponse'
+import getSpreeSdkMethodFromEndpointPath from '../../utils/get-spree-sdk-method-from-endpoint-path'
+import SpreeSdkMethodFromEndpointPathError from 'framework/spree/errors/SpreeSdkMethodFromEndpointPathError'
+import { GraphQLFetcher, GraphQLFetcherResult } from '@commerce/api'
+import createCustomizedFetchFetcher, {
+  fetchResponseKey,
+} from '../../utils/create-customized-fetch-fetcher'
+import fetch, { Request } from 'node-fetch'
```

Comment:
Can this be replaced with `@vercel/fetch` which is already included?

Reply:
We wanted to use `@vercel/fetch` but we had a problem with importing `Request` from there. We'll revisit this issue on Monday.

Reply:
@goncy 👋 I checked again. There are two issues with replacing `node-fetch`:
- We can't import the `Request` **class** from `@vercel/fetch`. The only export is the fetch setup function: 
![image](https://user-images.githubusercontent.com/1410097/144617484-3ef15880-01b3-45db-bc0d-631ff6e06ffe.png)
- We can't provide a `node-fetch` `Request` instance to `@vercel/fetch`. Doing the same with `node-fetch` works fine.

I suggest leaving this code as-is for now. WDYT?

Reply:
btw., There's an open issue covering this second point: https://github.com/vercel/fetch/issues/58